frontend/ui/lib/widgets/create_trip_page_widget.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:tripmeout/widgets/create_trip_widget.dart';
+import 'package:tripmeout/widgets/map_widget.dart';
+
+class CreateTripPageWidget extends StatelessWidget {
```

Comment:
Did you mean to replace this with `CreateTripWidget`? That would align your naming scheme with the view trip page (and also suggest that this widget might be embeddable in other pages). Also looks like `CreateTripWidget` isn't referenced anywhere right now.

Reply:
+1